checkov/terraform/graph_builder/utils.py

```diff
@@ -305,9 +305,6 @@ def get_file_path_to_referred_id_networkx(graph_object: DiGraph) -> dict[str, st
 
 def get_file_path_to_referred_id_igraph(graph_object: igraph.Graph) -> dict[str, str]:
     file_path_to_module_id = {}
-    for v in graph_object.vs:
-        if v[CustomAttributes.BLOCK_TYPE] == BlockType.MODULE:
-            modules = v
```

Comment:
why?

Reply:
Take a look on the following lines, they are overriding the var and doing exactly the same

Reply:
this looks redundant, when I see the line below 😄